build-system/app.js

```diff
@@ -857,6 +857,28 @@ app.use('/list/vegetable-data/get', (req, res) => {
   });
 });
 
+app.use('/subscription/subsplatform1', (req, res) => {
+  assertCors(req, res, ['GET']);
+  res.json({access: false});
```

Comment:
We'll always serve the format that you have below, e.g. `{entitlements: {source, products: [...]}}`. Notice that `service` probably won't be in the response ever - we just calculate it ourselves knowing which service we are using to fetch.